packages/nhsuk-frontend/src/nhsuk/components/checkboxes/template.njk

```diff
@@ -32,8 +32,9 @@
     {% set itemHintId = itemId ~ "-item-hint" if hasHint else "" %}
     {% set itemDescribedBy = describedBy if not hasFieldset else "" %}
     {% set itemDescribedBy = (itemDescribedBy ~ " " ~ itemHintId) | trim %}
+    {% set itemClasses = "nhsuk-checkboxes__input" ~ (" " ~ item.classes if item.classes) %}
     <div class="nhsuk-checkboxes__item">
-      <input class="nhsuk-checkboxes__input" id="{{ itemId }}" name="{{ itemName }}" type="checkbox" value="{{ item.value }}"
+      <input class="{{ itemClasses }}" id="{{ itemId }}" name="{{ itemName }}" type="checkbox" value="{{ item.value }}"
```

Comment:
`govuk-frontend` doesn't support `classes` on `checkbox.items[]`, but I think it’s a good thing to support.

However I wonder if you'd expect it to be added to the parent `.nhsuk-checkboxes__item` div instead?

Reply:
I've split out a PR to add `item.classes` etc if you'd like to comment there instead:

* https://github.com/nhsuk/nhsuk-frontend/pull/1683